README.md

```diff
@@ -44,3 +51,41 @@ In the project directory, run:
 * Builds the app for production to the `./build` folder
 * Correctly bundles React in production mode and optimizes the build for the best performance
 * Build is minified and filenames include hashes
+
+## Development
+
+### Git Workflow
+
+This project uses a [feature branch](http://nvie.com/posts/a-successful-git-branching-model/) based workflow.
+
+New features should be developed on new feature branches named in the following format: `[username]/[fancy-branch-name]`.
+Pull requests should be sent to the `develop` branch for code review.
```

Comment:
The development repo in `refinebio` is named "dev", can we rename "development" branch in this repo to "dev" to make it consistent?